_episodes/029-redisjson.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "RedisJson" 
+date: 2020-08-28T17:00:00+00:00file: 
```

Comment:
I'm guessing CI tests are failing because of the `file:` on the end here.  I assume this should be on a separate line with a URL?

Reply:
> What's the purpose of the `client_id` file that was added?

good question - I assumed it came from upstream after a `rebase`, I'll remove it.